acct-group/floppy/floppy-0.ebuild

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 Gentoo Authors
```

Comment:
Please move the addition of acct-group/floppy to a separate commit.

Also, you will need to create metadata.xml. You can set base-system@gentoo.org as the maintainer.